src/server/game/Scripting/ScriptReloadMgr.cpp

```diff
@@ -868,8 +859,6 @@ class HotSwapScriptReloadMgr final
 
         Optional<fs::path> cache_path;
```

Comment:
Is the optional still needed here? And also move initialise it from the expression below...